pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -663,13 +627,28 @@ class Build {
 
         context.node('worker') {
             context.stage("installer") {
-                // Ensure master context workspace is clean of any previous archives
-                context.sh "rm -rf workspace/target/* || true"
```

Comment:
These workspace clear need to remain. What's the point of move this clear to case " mac" and case " windows" separately ( Line 633 and Line 637)? @zdtsw

Reply:
no specific reason to have same line in both case.
unless we want to have regex to only delete certain file patterns in the future.